check_limits.py

```diff
@@ -1,18 +1,12 @@
-
-def battery_is_ok(temperature, soc, charge_rate):
-  if temperature < 0 or temperature > 45:
-    print('Temperature is out of range!')
-    return False
-  elif soc < 20 or soc > 80:
-    print('State of Charge is out of range!')
-    return False
-  elif charge_rate > 0.8:
-    print('Charge rate is out of range!')
-    return False
-
-  return True
-
-
+from Raise_Alert_If_Battery_Parameters_Exceeded_Limit import *
+
+limit = {
+     'temperature': {'min': 0, 'max': 45},
+     'state_of_charge': {'min': 20, 'max': 80},
+     'charge_rate': {'min' : 0, 'max': 0.8}
+        }     
+    
 if __name__ == '__main__':
-  assert(battery_is_ok(25, 70, 0.7) is True)
-  assert(battery_is_ok(50, 85, 0) is False)
+     raise_alert_battery_if_parameters_exceeded_limit({'temperature' : 25, 'state_of_charge' : 70, 'charge_rate' : 0.7})
```

Comment:
lower range breach is not tested?

Reply:
Lower breach is also tested but by mistake i named the function as "raise_alert_battery_if_parameters_exceeded_limit" which created the confusion. Now I have renamed the function as "raise_alert_battery_if_not_within_limit".